examples/service-clients/azure-client/todo-list/src/fluid.ts

```diff
@@ -0,0 +1,164 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type {
+	AzureLocalConnectionConfig,
+	AzureRemoteConnectionConfig,
+} from "@fluidframework/azure-client";
+// eslint-disable-next-line import/no-internal-modules -- #26985: `test-runtime-utils` internal used in example
+import { InsecureTokenProvider } from "@fluidframework/test-runtime-utils/internal";
+import {
+	type ContainerSchema,
+	type IFluidContainer,
+	SharedTree,
+	TreeViewConfiguration,
+} from "fluid-framework";
+import { SharedString } from "fluid-framework/legacy";
+import { v4 as uuid } from "uuid";
+
+import { AzureFunctionTokenProvider } from "./azureFunctionTokenProvider.js";
+import { TodoItem, TodoList } from "./schema.js";
+
+// Define the server we will be using and initialize Fluid
+const useAzure = process.env.FLUID_CLIENT === "azure";
+
+const user = {
+	id: uuid(),
+	name: uuid(),
+};
+
+const azureUser = {
+	id: user.id,
+	name: user.name,
+	additionalDetails: {
+		gender: "female",
+		email: "xyz@microsoft.com",
+	},
+};
+
+/**
+ * Azure Fluid Relay connection configuration.
+ */
+export const connectionConfig: AzureRemoteConnectionConfig | AzureLocalConnectionConfig =
+	useAzure
+		? {
+				type: "remote",
+				tenantId: "",
+				tokenProvider: new AzureFunctionTokenProvider("", azureUser),
+				endpoint: "",
+			}
+		: {
+				type: "local",
+				tokenProvider: new InsecureTokenProvider("fooBar", user),
+				endpoint: "http://localhost:7070",
+			};
+
+/**
+ * Schema for the TODO list application.
+ * @remarks
+ * This includes the DataObjects we support and any initial DataObjects we want created
+ * when the Container is first created.
+ */
+export const todoListContainerSchema = {
+	initialObjects: {
+		todoList: SharedTree,
+	},
+	dynamicObjectTypes: [SharedString],
```

Comment:
Since this is the core point of the demo, maybe consider elaborating here what is happening.